src/pages/loginPage.ts

```diff
@@ -0,0 +1,47 @@
+import {Page} from '@playwright/test';
+import PlaywrightWrapper from '../helper/wrapper/playwrightWrappers';
+import Assert from '../helper/wrapper/assert';
+
+export default class LoginPage {
+  private base: PlaywrightWrapper;
+  private assert: Assert;
+
+  constructor(private page: Page) {
+    this.base = new PlaywrightWrapper(page);
+  }
+
+  public Elements = {
+    logInBtn: "//a/span[text()='Log in']",
+    emailInput: "//input[@name='email']",
+    passwordInput: "//input[@name='password']",
+    submitBtn: "//button[@type='submit']"
+  };
+
+  async clickLogInButton() {
+    this.page.waitForTimeout(2000);
```

Comment:
It's not good idea to use explicit waits until it's necessary.

Reply:
It's necessary